pycbc/transforms.py

```diff
@@ -2501,3 +2501,51 @@ def read_transforms_from_config(cp, section="transforms"):
         t = transforms[name].from_config(cp, section, subsection)
         trans.append(t)
     return order_transforms(trans)
+
+
+def new_z_to_euler(new_z):
+    """
+    From the new Z axis expressed in (polar, azimuthal) angles of the
+    initial coordinate system, return the (alpha, beta) Euler angles
+    that rotate the old Z axis (0, 0) to the new Z axis.
+    """
+    return (lal.PI_2 + new_z[1]) % (2 * lal.PI), new_z[0]
```

Comment:
I suggest using `numpy.pi` rather than `lal.PI` and `lal.PI_2`.

Reply:
The suggested change is made